example/.snyk

```diff
@@ -0,0 +1,25 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```

Comment:
I don't think we actually *can* do this without a ticket/etc.  See cumulus-cicd for more on that point.

Reply:
Thanks. It's checked in by accident.